gammapy/scripts/spectrum_pipe_example.yaml

```diff
@@ -0,0 +1,36 @@
+# Example Gammapy spectrum analysis pipeline config file
+
+general:
+  datastore: /home/kingj/Work/HESSData
+#  observations : [23526, 23559]
+  observations : crab_4tel.lis
+
+binning:
+  equal_log_spacing: true
+  emin: 0.1
+  emax: 10
+  unit: TeV 
+  nbins: 30
+
+on_region:
+  type: circle
+  center_x: 83.633083
+  center_y: 22.014500
+  unit: deg
+  system: icrs
+  radius: 0.1
+
+off_region:
+  type: ring
+  inner_radius: 0.5
+  outer_radius: 0.6
+
+ogip:
+  clobber: true
+
+model:
+   type: PWL
+   threshold_low: 0.1
```

Comment:
Is it possible / better to have the unit on the same line, i.e. have the string "0.1 TeV" here that then gets passed to the Quantity constructor?
If this doesn't work out of the box or you don't think it's better that way, OK to leave as is.